spiffworkflow-frontend/test/browser/tasks/public_tasks/test_can_start_process_from_message_form.py

```
@@ -0,0 +1,27 @@
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)